src/wp-admin/customize.php

```diff
@@ -84,18 +84,20 @@
 	}
 }
 
-$url       = ! empty( $_REQUEST['url'] ) ? sanitize_text_field( $_REQUEST['url'] ) : '';
-$return    = ! empty( $_REQUEST['return'] ) ? sanitize_text_field( $_REQUEST['return'] ) : '';
-$autofocus = ! empty( $_REQUEST['autofocus'] ) ? sanitize_text_field( $_REQUEST['autofocus'] ) : '';
+$url    = ! empty( $_REQUEST['url'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['url'] ) ) : '';
+$return = ! empty( $_REQUEST['return'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['return'] ) ) : '';
+$autofocus = ! empty( $_REQUEST['autofocus'] ) && is_array( $_REQUEST['autofocus'] )
+	? array_map( 'sanitize_text_field', wp_unslash( $_REQUEST['autofocus'] ) )
+	: array(); // Since `$_REQUEST['autofocus']` is an array.
```

Comment:
```suggestion
	: array(); // `$_REQUEST['autofocus']` is an array.
```
"Since" could be confused with `@since`, so this word might be omitted, or changed to "Because" to remove [ambiguity](https://grammarbrain.com/because-vs-since/).

Reply:
Good point @ironprogrammer.

Thinking this inline comment isn't necessary, as it's self-explanatory. 

It's setting a default value of an empty `array`. When evaluating the `true` branch, an `array` will be assigned to `$autofocus` (via `array_map()` - see [PHP docs](https://www.php.net/manual/en/function.array-map.php)). Given that context and the default value, I'm thinking it's self-explanatory.